modules/ROOT/pages/_partials/api-mgr.adoc

```diff
@@ -59,8 +59,7 @@ Besides the default `--help`, general, and authentication flags, this command al
 Supported values: days, hours, minutes.
 | `--email <emailAddress>` | Email to send alert notification to. +
 You can pass this flag multiple times to specify multiple emails.
-| `--enabled` | Sets whether the alert should be enabled or not. +
-Possible values are `true` or `false`.
+| `--enabled` | Sets whether the alert should be enabled or not. + Include the flag to enable it.
```

Comment:
```suggestion
| `--enabled` | Sets whether the alert should be enabled or not. Include the flag to enable it.
```

Reply:
The way this was, the plus sign appears in the content. If you want Include the flag to enable it to be on the next line, you'll need to leave the plus sign as it was and put that text on the next line as the line you deleted was before.